project/MiMaFilters.scala

```diff
@@ -0,0 +1,13 @@
+
+import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.core.ProblemFilters._
+
+object MiMaFilters {
+  val Library: Seq[ProblemFilter] = Seq(
+    // New APIs marked @experimental in 3.0.1
+    exclude[DirectMissingMethodProblem]("scala.quoted.Quotes#reflectModule#TermParamClauseMethods.isErased"),
+    exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#TermParamClauseMethods.isErased"),
+    exclude[MissingClassProblem]("scala.annotation.internal.ErasedParam"),
+    exclude[MissingClassProblem]("scala.annotation.internal.Stable"),
```

Comment:
This class doesn't exist anymore.

Reply:
Also isn't this missing experimental itself?

Reply:
Yes